lib/rubocop/cop/sorbet/binding_constants_without_type_alias.rb

```diff
@@ -67,9 +84,17 @@ def on_casgn(node)
 
         def autocorrect(node)
           lambda do |corrector|
+            if using_deprecated_type_alias_syntax?(node)
+              corrector.replace(
```

Comment:
Sorbet itself has an autocorrect for this that picks the `do; end` syntax for multi-line aliases. Maybe we can leave this to Sorbet?

Reply:
I was wondering the same thing. I wanted to have a custom error for the old syntax and while at it I was able to give the autocorrect.

I can remove the autocorrect but I would like to keep the custom message as the current want is a bit misleading (see #15).

Reply:
I think keeping an autocorrect here is a good idea. Sorbet doesn't let you selectively autocorrect (as far as I know), and in my experience running `srb tc -a` sometimes gets false positives. Keeping this will allow you to autocorrect just this one issue.